llvm/test/ThinLTO/X86/coro.ll

```diff
@@ -0,0 +1,21 @@
+; RUN: llvm-as %s -o %t1.bc
```

Comment:
This looks like it would be regular LTO not ThinLTO as the path of the test implies. This is not going to generate a ThinLTO summary. Is this test meant to go in llvm/test/LTO/X86 instead? Also, it looks like this code change shouldn't affect ThinLTO.

Reply:
We compile some objects during LTO indexing through this function.
Didn't touch that code for a while but it looks like `lto_unit` stuff for CFI

Reply:
In the test I don't try to replicate thinlto,
just to make sure that buildLTODefaultPipeline do not left coro unprocessed.

Reply:
I'm not sure what you mean. We can't do any ThinLTO without a module summary. llvm-as does not create a summary. Only something like "opt --thinlto-bc"

Reply:
> In the test I don't try to replicate thinlto, just to make sure that buildLTODefaultPipeline does not left coro unprocessed.

Right, that's the regular LTO backend. So I suggest moving this to "llvm/test/LTO/X86". I.e. there is no ThinLTO triggered by this test, and the PR does not affect ThinLTO handling anyway.

Reply:
Done!